core-services/prow/02_config/_plugins.yaml

```diff
@@ -1496,6 +1512,31 @@ plugins:
   - owners-label
   - wip
   - yuks
+  openshift-priv:
```

Comment:
This should have BZ -- CVE also have BZ

Reply:
It's not necessary for the spike and I'm expecting BZ config to be "special" for openshift-priv, so I'm trying to not pull it in for now.

Reply:
I think it will be identical ...